books/RayTracingInOneWeekend.html

```diff
@@ -1942,99 +2048,29 @@
 
-  1. The shadows are less pronounced after the change
-  2. Both spheres are lighter in appearance after the change
+  1. The shadows are more pronounced after the change
+  2. Both spheres are darker in appearance after the change
 
-Both of these changes are due to the more uniform scattering of the light rays, fewer rays are
-scattering toward the normal. This means that for diffuse objects, they will appear _lighter_
-because more light bounces toward the camera. For the shadows, less light bounces straight-up, so
-the parts of the larger sphere directly underneath the smaller sphere are brighter.
-</div>
-
-
-An Alternative Diffuse Formulation
------------------------------------
-<div class='together'>
-The initial hack presented in this book lasted a long time before it was proven to be an incorrect
-approximation of ideal Lambertian diffuse. A big reason that the error persisted for so long is
-that it can be difficult to:
+Both of these changes are due to the less uniform scattering of the light rays, more rays are
+scattering toward the normal. This means that for diffuse objects, they will appear _darker_
+because fewer light bounces toward the camera. For the shadows, more light bounces straight-up, so
+the parts of the larger sphere directly underneath the smaller sphere are darker.
 
-  1. Mathematically prove that the probability distribution is incorrect
-  2. Intuitively explain why a $\cos (\phi)$ distribution is desirable (and what it would look like)
 
 Not a lot of common, everyday objects are perfectly diffuse, so our visual intuition of how these
-objects behave under light can be poorly formed.
-</div>
-
-<div class='together'>
-In the interest of learning, we are including an intuitive and easy to understand diffuse method.
-For the two methods above we had a random vector, first of random length and then of unit length,
-offset from the hit point by the normal. It may not be immediately obvious why the vectors should be
-displaced by the normal.
-
-A more intuitive approach is to have a uniform scatter direction for all angles away from the hit
-point, with no dependence on the angle from the normal. Many of the first raytracing papers used
-this diffuse method (before adopting Lambertian diffuse).
-
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-    vec3 random_in_hemisphere(const vec3& normal) {
-        vec3 in_unit_sphere = random_in_unit_sphere();
-        if (dot(in_unit_sphere, normal) > 0.0) // In the same hemisphere as the normal
-            return in_unit_sphere;
-        else
-            return -in_unit_sphere;
-    }
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [random-in-hemisphere]: <kbd>[vec3.h]</kbd> The random_in_hemisphere(normal) function]
-
+objects behave under light can be poorly formed. Scenes will become more complicated over the course
```

Comment:
“As scenes become more complicated over the course of the book, you are encouraged ...”